clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java

```diff
@@ -388,6 +388,47 @@ public AlterConfigsResult alterConfigs(Map<ConfigResource, Config> configs) {
      */
     public abstract AlterConfigsResult alterConfigs(Map<ConfigResource, Config> configs, AlterConfigsOptions options);
 
+    /**
+     * Incrementally updates the configuration for the specified resources with default options.
+     *
+     * This is a convenience method for #{@link AdminClient#alterConfigs(Collection, IncrementalAlterConfigsOptions)} with default options.
+     * See the overload for more details.*
+     *
+     * This operation is supported by brokers with version 2.3.0 or higher.
+     *
+     * @param configs         The resources with their configs
+     * @return                The IncrementalAlterConfigsResult
+     */
+    public IncrementalAlterConfigsResult alterConfigs(Collection<AlterConfigOp> configs) {
```

Comment:
Can we have API like  ```alterConfigs(Map<ConfigResource, Collection<AlterConfigOp>> configs.```. 
This will be similar to existing ```alterConfigs(Map<ConfigResource, Config> configs)``` API.

Reply:
I think that's what I suggested in the KIP and Colin eventually agreed.

Reply:
Updated the API. Will update the KIP after the review.